pyfixest/utils.py

```diff
@@ -35,80 +36,101 @@ def ssc(adj=True, fixef_k="none", cluster_adj=True, cluster_df="min"):
     if cluster_df not in ["conventional", "min"]:
         raise ValueError("cluster_df must be 'conventional' or 'min'.")
 
-    res = {
+    return {
         "adj": adj,
         "fixef_k": fixef_k,
         "cluster_adj": cluster_adj,
         "cluster_df": cluster_df,
     }
 
-    return res
-
 
 def get_ssc(ssc_dict, N, k, G, vcov_sign, vcov_type, is_twoway=False):
     """
-    Compute small sample adjustment factors
-
-    Args:
-    - ssc_dict: An dictionariy created via the ssc() function
-    - N: The number of observations
-    - k: The number of estimated parameters
-    - G: The number of clusters
-    - vcov_sign: A vector that helps create the covariance matrix
-    - vcov_type: Either "iid", "hetero" or "CRV"
-    - is_twoway: Whether the covariance matrix is of the form V = V_1 + V_2 - V_12, i.e. whether it is a two-way cluster matrix
-
-    Returns:
-    - A small sample adjustment factor
+    Compute small sample adjustment factors.
+
+    Parameters
+    ----------
+    ssc_dict : dict
+        A dictionary created via the ssc() function.
+    N : int
+        The number of observations.
+    k : int
+        The number of estimated parameters.
+    G : int
+        The number of clusters.
+    vcov_sign : array-like
+        A vector that helps create the covariance matrix.
+    vcov_type : str
+        The type of covariance matrix. Must be one of "iid", "hetero", or "CRV".
+    is_twoway : bool, optional
+        Whether the covariance matrix is of the form V = V_1 + V_2 - V_12. Default is False.
+
+    Returns
+    -------
+    float
+        A small sample adjustment factor.
+
+    Raises
+    ------
+    ValueError
+        If vcov_type is not "iid", "hetero", or "CRV", or if cluster_df is neither "conventional" nor "min".
     """
-
     adj = ssc_dict["adj"]
-    fixef_k = ssc_dict["fixef_k"]
+    fixef_k = ssc_dict["fixef_k"]  # noqa: F841 TODO: is this used?
```

Comment:
This is not yet used and still needs to be implemented. This is the main reason why pyfixest standard errors slightly differ from fixest standard errors when there are fixed effects (nested within clusters).